src/coreclr/debug/daccess/enummem.cpp

```diff
@@ -1558,6 +1558,33 @@ HRESULT ClrDataAccess::EnumMemCLRMainModuleInfo()
         status = E_UNEXPECTED;
     }
 
+    if (pe.HasDirectoryEntry(IMAGE_DIRECTORY_ENTRY_EXPORT))
+    {
+        COUNT_T size = 0;
+        TADDR pExportTablesOffset = pe.GetDirectoryEntryData(IMAGE_DIRECTORY_ENTRY_EXPORT, &size);
+
+        ReportMem(pExportTablesOffset, size, true);
+
+        // This following part is a bit manual. We have to do this as we generate this
+        // structure from the DAC binary itself, so we have a chicken and egg problem.
+        // Instead we break the cycle by hardcoding the size  of the struct and assert
+        // the size of the definition with a warning to update this side. Only single
+        // files will have the runtime info struct.
+        const LPCSTR RuntimeInfoExportName = "DotNetRuntimeInfo";
+        const UINT32 RuntimeInfoExportSize = 96;
```

Comment:
Can't we put these defines in runtimeinfo.h? I guess I don't understand this cycle.  "runtimeinfo.h" isn't generated.